INSTALL.md

```diff
@@ -1,68 +1,54 @@
 
-After installation, we can use pyenv to install Python from within the repo.
 ```bash
 cd <repo_location>
-pyenv install 3.9
-pyenv local 3.9
+pyenv install 3.10
+pyenv local 3.10
 python -m venv .venv
 source .venv/bin/activate
 ```
-Once you're in your virtual environment, install the project:
+
+### 4. Install Agent0
+
+All individual subpackages can be installed from the outermost repo through `requirements.txt`
 
 ```bash
 python -m pip install --upgrade pip
 python -m pip install -r requirements.txt
 python -m pip install -r requirements-dev.txt
 ```
 
-- `pyenv install 3.9` You should now see the correct version when you run `pyenv versions`.
-- `pyenv local 3.9` This command creates a `.python-version` file in your current directory. If you have pyenv active in your environment, this file will automatically activate this version for you.
+An explanation of what the above steps do:
+- `pyenv install 3.10` You should now see the correct version when you run `pyenv versions`.
+- `pyenv local 3.10` This command creates a `.python-version` file in your current directory. If you have pyenv active in your environment, this file will automatically activate this version for you.
 - `python -m venv .venv` This will create a `.venv` folder in your repo directory that stores the local python build & packages. After this command you should be able to type which python and see that it points to an executable inside `.venv/`.
 - `python -m pip install -r requirements.txt` This installs the subpackages locally such that the install updates automatically any time you change the source code.
 - `python -m pip install -r requirements-dev.txt` installs dev related packages such as pytest.
 
-Finally, you can test that everything is working by calling:
-`PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 python -m pytest tests/`
-
-## Apeworks and Contract Integration
-
-We run several tests and offer utilities that depend on executing Hyperdrive solidity contracts. This is not required to use elfpy.
-
-NOTE: The Hyperdrive solidity implementation is currently under security review, and thus is not available publicly.
-The following instructions will not work for anyone who is not a member of Delv.
-
-First, [install Forge](https://github.com/foundry-rs/foundry#installatio://github.com/foundry-rs/foundry#installation).
-
-Next, to use apeworx with elfpy, clone and sym link the hyperdrive repo, into `hyperdrive_solidity/`, i.e.:
-
-```bash
-git clone https://github.com/delvtech/hyperdrive.git ../hyperdrive
-ln -s ../hyperdrive hyperdrive_solidity
-```
-
-then run:
-
+Alternatively, each subpackage in `lib` can be individaully installed through pip. For example, you can install `fixedpointmath` through pip as follows:
 ```bash
-ape plugins install .
-ape compile
+python -m pip install lib/fixedpointmath # Install from local file
+# TODO add installation through git here
```

Comment:
Might as well add that in if you think we should include it. Example syntax for elf-simulations:
`pip install git+https://github.com/delvtech/elf-simulations`

Reply:
I think it's nontrivial to install via git from subdirectory, need this PR to be merged for testing